### PR TITLE
[bitnami/wordpress] Chart standardized

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juanariza@vmware.com>

**Description of the change**

This PR standardizes the `bitnami/wordpress` chart to be in line with the rest of the catalog.

**Benefits**

Standardization.

**Possible drawbacks**

Changes the values structure forcing a major bump, check the README.md for more information.

**Applicable issues**

N/A

**Additional information**

Blocked by https://github.com/bitnami/charts/pull/8668 (major bump in dep)

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/wordpress/values.yaml [hunk 1]
```diff
@@ -786,40 +880,51 @@ metrics:
...
+    ## @param metrics.serviceMonitor.scrapeTimeout Timeout after which the scrape is ended
+    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
     ##
     scrapeTimeout: ""
-    ## @param metrics.serviceMonitor.relabellings Metrics relabellings to add to the scrape endpoint
+    ## @param metrics.serviceMonitor.additionalLabels Additional labels that can be used so ServiceMonitor will be discovered by Prometheus
     ##
-    relabellings: []
-    ## @param metrics.serviceMonitor.honorLabels Labels to honor to add to the scrape endpoint
+    additionalLabels: {}
```
Comment (migruiz4) [inline bitnami/wordpress/values.yaml:911]: Please use `metrics.serviceMonitor.labels` instead.

**Review by migruiz4 — APPROVED** [review]
Approved but please check my comment below.

**Comment by migruiz4** [comment]
Additionally, make sure to run `dep update` before merging.

File: bitnami/wordpress/templates/ingress.yaml [hunk 2]
```diff
@@ -46,18 +43,15 @@ spec:
...
-  {{- if or .Values.ingress.tls .Values.ingress.extraTls }}
+  {{- if or (and .Values.ingress.tls (or (include "wordpress.ingress.certManagerRequest" .Values.ingress.annotations) .Values.ingress.selfSigned)) .Values.ingress.extraTls }}
   tls:
-    {{- if .Values.ingress.tls }}
+    {{- if and .Values.ingress.tls (or (include "wordpress.ingress.certManagerRequest" .Values.ingress.annotations) .Values.ingress.selfSigned) }}
     - hosts:
         - {{ .Values.ingress.hostname | quote }}
-        {{- range .Values.ingress.extraHosts }}
-        - {{ .name | quote }}
-        {{- end }}
```
Comment (weeix) [inline bitnami/wordpress/templates/ingress.yaml:56]: @juan131 Hi, I noticed that this PR reverses the changes in https://github.com/bitnami/charts/pull/4953. Could you please explain why this is necessary?

**Pull request merged**